build.sh

```diff
@@ -100,5 +100,14 @@ for dir in ${dirs}; do
     fi
   fi
 
+  if [[ -v TEST_OPENSHIFT_MODE ]]; then
+    if [[ -x test/run-openshift ]]; then
+      VERSION=$dir IMAGE_NAME=${IMAGE_NAME} test/run-openshift
+    else
+      echo "-> OpenShift tests are not present, skipping"
+    fi
+  fi
+
+
```

Comment:
What I dislike about build.sh script is that it does both building and testing.  And now it would serve another task.  Such things are hard to be parallelized in future.

It would be much nicer if we had self-standing script for this without the need for  TEST_OPENSHIFT_MODE variable.